web-frontend/src/pages/TransferCoins/SendCoins/SendConfirmation.tsx

```diff
@@ -39,9 +39,33 @@ export function SendConfirmation({ ...props }) {
     mutate(transferData as SendTransactionObject);
   };
 
+  const [customFees, setCustomFees] = useState<string>('Standard');
+
+  useEffect(() => {
+    switch (fees) {
+      case 1000:
+        setCustomFees('Standard');
+        break;
+      case 1:
+        setCustomFees('Low');
+        break;
+      case 5000:
+        setCustomFees('High');
+        break;
+      default:
+        setCustomFees('Standard');
+    }
+  }, [fees]);
+
+  const content = `
+  ${Intl.t('sendcoins.gas-info', {
+    default: customFees,
+    xx: fees,
```

Comment:
mayme rename xx ?

Reply:
> maybe rename xx in the i18 ?

@pivilartisant Yes! You can change to `nbFees`. 
Or other explicit name.

Reply:
set to gasFees